src/binding/fortran/use_mpi_f08/mpi_c_interface_nobuf.f90

```diff
@@ -1384,15 +1384,15 @@ function MPIR_Add_error_string_c(errorcode, string) &
 end function MPIR_Add_error_string_c
 
 function MPIR_Delete_error_class_c(errorclass) &
-    bind(C, name="PMPIX_Delete_error_class") result(ierror)
+    bind(C, name="PMPI_Delete_error_class") result(ierror)
```

Comment:
These delete functions are not in MPI-4.0. That should retain the X for now.

Reply:
Okay

Reply:
In this case, I'll close this PR and re-introduce the comm commits in the split comm api autogen PR.